membership/models.py

```diff
@@ -933,7 +933,7 @@ class Meta:
         permissions = (
             ("can_import_payments", "Can import payment data"),
         )
-
+        unique_together = ('reference_number', 'transaction_id', 'message')
```

Comment:
Based on the original issue description, I'd have thought it'd be better to unique transaction_id and payment_day, not reference number and message?

I don't know why we have payment_day is a DateTimeField, and not just a date, though.

Reply:
That is true, fixed in newest commit.